app/controllers/admins/admins_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Admins
+  class AdminsController < ApplicationController
+    before_action :admin
```

Comment:
not `admin`, just `current_admin`

Reply:
`current_admin` is not available because authentication hasn`t implemented yet